src/proto/graplinc/grapl/api/graph_query_proxy/v1beta1/graph_query_proxy.proto

```diff
@@ -0,0 +1,51 @@
+syntax = "proto3";
+
+package graplinc.grapl.api.graph_query_proxy.v1beta1;
+
+import "graplinc/grapl/api/graph_query/v1beta1/graph_query.proto";
+import "graplinc/grapl/common/v1beta1/types.proto";
+
```

Comment:
## These 4 messages exactly the same as the ones in `graph_query.proto` except for the removal of `tenant_id` in the requests.